index.html

```diff
@@ -0,0 +1,89 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="author" content="Nemwel Boniface">
+    <meta name="description" content="This is my personal portfolio site that I will share to all potential employers to showcase my skills as a developer">
+
+    <!--google Fonts roboto, crete-round-->
+    
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&display=swap" rel="stylesheet">
+
+    <!-- font awesome icons -->
+
+    <link rel="stylesheet" href="http://maxcdn.bootstrapcdn.com/font-awesome/4.2.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+
+    <title>Nemwel Portfolio</title>
+
+    <!--Page stylesheets-->
+    <link rel="stylesheet" href="stylesheets/landing.css">
+
+    <!--Smooth scroll script-->
+       
+    <script src="https://cdn.jsdelivr.net/gh/cferdinandi/smooth-scroll/dist/smooth-scroll.polyfills.min.js"></script>
+
+</head>
+<body>
+    <header class="header">
+        <div class="topHeaderBar">
+            <div class="time-left">
+                <h2>9:41</h2>
+            </div>
+
+            <div class="icons-right">
+                <i class="fa fa-signal" aria-hidden="true"></i>
+                <i class="fa fa-wifi" aria-hidden="true"></i>
+                <i class="fa fa-battery-full" aria-hidden="true"></i> 
+            </div>
+        </div>
+
+        <nav class="topnavBar">
+            <div class="navleft">
+                <h2><a href="#">Nemwel Boniface</a></h2>
+            </div>
+
+            <nav class="navright">
+                <i class="fa fa-bars" aria-hidden="true"></i>
+            </nav>
+        </nav>
+    </header> 
+
+    <main class="main">
+        <section class="authordetails">
+            <div class="authorheader">
+                <h2>Hey There. I'm Nemwel</h2>
```

Comment:
- [OPTIONAL]

- [ ] I suggest you change this `h2` tag to `h1` tag as it is the main title of your page and therefore should be wrapped with the `h1` tag. This is to improve the structure of your HTML and make your code more professional.